From 719f1ce5fb41493fe10edca1ba9223fa601b6165 Mon Sep 17 00:00:00 2001
From: Hendrik Leppkes <h.leppkes@gmail.com>
Date: Thu, 12 Dec 2013 21:12:51 +0100
Subject: [PATCH] vc1: set chromaformat = 1 for simple/main profile

1 is the only valid value for VC-1/WMV3, and setting it here makes sure
no invalid value is send to a hw accelerator, for example.

Signed-off-by: Michael Niedermayer <michaelni@gmx.at>
---
 xbmc/lib/ffmpeg/libavcodec/vc1.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/xbmc/lib/ffmpeg/libavcodec/vc1.c b/xbmc/lib/ffmpeg/libavcodec/vc1.c
index fb33e6f..f8d3162 100644
--- a/xbmc/lib/ffmpeg/libavcodec/vc1.c
+++ b/xbmc/lib/ffmpeg/libavcodec/vc1.c
@@ -293,6 +293,7 @@ int ff_vc1_decode_sequence_header(AVCodecContext *avctx, VC1Context *v, GetBitCo
         v->zz_4x8 = ff_vc1_adv_progressive_4x8_zz;
         return decode_sequence_header_adv(v, gb);
     } else {
+        v->chromaformat = 1;
         v->zz_8x4 = ff_wmv2_scantableA;
         v->zz_4x8 = ff_wmv2_scantableB;
         v->res_y411   = get_bits1(gb);
-- 
1.8.5.1

